Ginger/Ginger/SourceControl/SourceControlProjectsPage.xaml.cs

```diff
@@ -96,6 +107,16 @@ private void Bind()
             App.ObjFieldBinding(SourceControlURLTextBox, TextBox.TextProperty, mSourceControl, SourceControlBase.Fields.SourceControlURL);
             App.ObjFieldBinding(SourceControlUserTextBox, TextBox.TextProperty, mSourceControl, SourceControlBase.Fields.SourceControlUser);
             App.ObjFieldBinding(SourceControlPassTextBox, TextBox.TextProperty, mSourceControl, SourceControlBase.Fields.SourceControlPass);
+            if (App.UserProfile.SourceControlType == SourceControlBase.eSourceControlType.GIT)
+            {
+                txtConnectionTimeout.Visibility = Visibility.Hidden;
```

Comment:
why we have visibility code in 2 places Init and Bind. See if need it at 2 places. Also add it to panel and set visibility for panel instead of individual component